### ISSUE TITLE
Error: bad track: 'mett' bad track: 'mett'

### ISSUE THREAD

**Issue by SeveX** [anthwlock/untrunc#143]
Hello,

I hope you're doing well today. I get the following error message whenever I try to restore my video:

![image](https://github.com/anthwlock/untrunc/assets/39397656/a8cae58f-29a5-4f5f-b271-654c4eedab78)

Please let me know if there is a way to resolve this error.

Thank you!

**Comment by Finni123** [comment]
I get the same error, did you find a solution?

**Comment by SeveX** [comment]
> I get the same error, did you find a solution?

Nope. I was lucky enough to find the original recorded video on my phone. I hope the developer is able to help though.
